recipe/meta.yaml

```diff
@@ -27,12 +22,13 @@ source:
     - 0012-macOS-include-cairo_h-not-cairo-xlib_h.patch
 
 build:
-  number: 1
+  number: 2
   rpaths:
     - lib/R/lib/
     - lib/
   detect_binary_files_with_prefix: true
-  skip: true  # [win32 or linux32]
+  # No openjdk on win32.
+  skip: true  # [win32]
```

Comment:
I did get a successful build in Linux-32, either way conda-forge only builds Linux-64. I also added a comment on the main reason I found for not trying a Win-32 build.